spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Utils.scala

```diff
@@ -25,13 +25,19 @@ import com.intel.analytics.bigdl.utils.T
 
 private[bigdl] object Utils {
   def copyMaskAndScales(from: MemoryData, to: MemoryData): Unit = {
-    if (to.scales.isEmpty) {
+    // here we check the from and to, they should not be null ideally
+    // but if the model is mixin with blas layer, it may be null
+    if (from != null && to != null && to.scales.isEmpty) {
```

Comment:
what will happen if "from" has empty scales?

Reply:
Nothing. The  code is be compatible with empty from scales.